jax/experimental/jax2tf/README.md

```diff
@@ -1005,12 +1005,27 @@ The `jax2tf`-lowered function supports higher-order gradients, but when the
 function is saved in a SavedModel, only the first-order gradient is saved.
 This is primarily a limitation of the SavedModel support for custom gradients.
 
+### Native serialization supports only select dialects
+
+Applies to native serialization only.
+
+JAX native serialization checks that the code to be serialized contains
+operations only from MLIR dialects that are known to have stability guarantees,
+e.g., StableHLO.
+
+If your code uses custom JAX lowering rules that use operations from other
+dialects, and if you have reasons to believe that those operations are stable
+over the planned lifetime of the serialized artifact,
```

Comment:
How does one determine the planned lifetime? Is it specific to a particular artifact, or there's some overall standard?

Reply:
This is meant to cover exceptional situations where you know that you do not need stability more than a certain window, e.g., because you know exactly how and when an artifact will be used. Or a team has a custom primitive lowered to a custom call that they own and they are sure the custom call target will be stable.